redis_lock/asyncio/types.py

```diff
@@ -0,0 +1,8 @@
+from datetime import timedelta
+from typing import Union
+
+from redis.asyncio import Redis as AsyncRedis
+
+RedisClient = AsyncRedis
+LockKey = Union[bytes, str]
+TimeOutType = Union[int, timedelta]
```

Comment:
These type definitions don’t appear to be used anywhere.
Since same types are already managed in `redis_lock/types.py`, please consider removing them.